guidelines/content-guidelines/content-strategy.md

```diff
@@ -39,22 +39,31 @@ Every independent Kyma component is a separate documentation topic. The only exc
 
 ## Documentation types
 
+There is a set of documents that a given technical topic must include. You can also add additional document types to expose a specific topic better. However, before you create an additional document type, consult it with other Kyma content developers.  
```

Comment:
Maybe mentioning a SIG Content would be a good idea? We could also provide a link to SIG Content README so that sb who'd like to introduce changes could see when our meetings take place.

Reply:
Already discussed and clarified that.